docs/getting-started/getting-started-with-galaxy.rst

```diff
@@ -2,16 +2,22 @@
 Getting Started with Galaxy
 ===========================
 
-Galaxy workflow support in Dockstore is new and a preview of
-more full featured support to come.
+Dockstore supports Galaxy workflows and the :doc:`Launch with Galaxy <../../launch-with/galaxy-launch-with>` tutorial elaborates different ways to launch a Galaxy workflow. Please note that Galaxy workflows cannot be launched by the Dockstore CLI.
+
+Galaxy instances are offered by many institutions, free and otherwise. Each come with their own advantages and users can make a choice based on their own needs.
+
+Using Galaxy Servers
+--------------------
+Institutions across the world maintain Galaxy instances that are often shared with the greater community for free, subsidized by grants, such as the `Use Galaxy Org <https://usegalaxy.org/>`. These are great ways to learn and use Galaxy, and many offer valuable learning resources such as the `Galaxy Training Network <https://training.galaxyproject.org/>`.
```

Comment:
Both the links do not show up correctly on https://docs.dockstore.org/en/feature-update-launchwithgalaxy/getting-started/getting-started-with-galaxy.html.

I think you need trailing double underscores on each link, but please verify.